index.js

```diff
@@ -83,15 +85,25 @@ const MultiselectPrompt = {
 		this.render()
 	}
 	, right: function () {
-		this.value[this.cursor].selected = true
-		this.render()
+		if (this.value.filter (e => e.selected).length < this.maxChoices) {
```

Comment:
Ho okay, this is purely syntactical right?
Because this is the exact same code otherwise. I'll change it, no problem.